qemu/tests/set_link.py

```diff
@@ -154,6 +155,7 @@ def set_link_test(linkid, link_up, expect_status,
 
         """
         vm.set_link(linkid, up=link_up)
```

Comment:
why not sleep in  vm.set_link(), maybe this issue will happen in other case too.

Thanks,
@xutian